modules/serverless-pingsource-kn.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+[id="serverless-pingsource-kn_{context}"]
+= Using PingSource with the `kn` CLI
+
+The following sections describe how to create, verify and remove a basic PingSource using the `kn` CLI.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+
+. For each set of ping events that you want to request, create a PingSource in the same namespace as the event consumer:
++
+----
+$ kn source ping create test-ping-source --schedule "*/2 * * * *" --data '{"message": "Hello world!"}' --sink svc:event-display
```

Comment:
Here this is even linger line that after docs are built we have horizontal slider in code box. We can write that as:

```adoc
----
$ kn source ping create test-ping-source \
    --schedule "*/2 * * * *" \
    --data '{"message": "Hello world!"}' \
    --sink svc:event-display
----
```